public/project/css/landing.css

```diff
@@ -0,0 +1,61 @@
+/* Landing Container */
+.landing-container {
+  background: url('http://www.wall321.com/thumbnails/detail/20120308/cityscapes%20city%20cars%20aston%20martin%20james%20bond%20bridges%20silver%20cars%20dbs%201920x1200%20wallpaper_www.wall321.com_15.jpg') no-repeat center center fixed;
+  background-size: cover !important;
+  background-position: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  box-shadow: 0 0 85px black;
+}
+
+/* Landing Overlay */
+.landing-overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(1, 1, 1, 0.7);
+}
+
+/* Landing Center Content Container */
+.landing-content {
+  width: 400px;
+  height: 200px;
+  max-width: 100%;
+}
```

Comment:
So max-width 100% makes no sense, since 100% is the maximum width lol. So what you can do is, set the width to 100%, and then set the max-width to 400px. It is also reccommended to set the min-width. Usually 280px is the perfect width - because the smallest width you will have to deal with is 320px, which is the width of an iphone. So 280px is the perfect width for mobile.